static/examples/hashing-tool/index.html

```diff
@@ -0,0 +1,167 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <title>UID2 Hashing Tool</title>
+    <link rel="stylesheet" type="text/css" href="./app.css" />
+    <link rel="shortcut icon" href="/img/favicon.ico" />
+    <!-- <script src="https://cdn.integ.uidapi.com/uid2-sdk-3.2.0.js"></script> -->
```

Comment:
I think it probably makes good sense to have a local version of the js sdk here.  so you can delete the commented lines.